OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -4,13 +4,15 @@
 
 iq:
   name: nxiq
-  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server
-  imageTag: 1.85.0-01-ubi
+  imageName: sonatype/nexus-iq-server
```

Comment:
we had used the RH version because that was a requirement for getting this chart certified. Why are you changing this? It's in the values file so if you need to override it, you can but I think we should stick with the RH as a default.

Reply:
Agreed in the context of an Operator but we chose to use helm so we also get helm charts of this for generic k8s instances